src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -22,6 +22,11 @@ internal static class WatsonDisabled
     internal static class WatsonReporter
     {
         /// <summary>
+        /// hold onto last issue we reported
+        /// </summary>
+        private static string s_lastExceptionReported;
```

Comment:
Can the exception string be very long, if we have lot of exception nesting? Consider using the hash instead.

Reply:
good idea.